app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -148,8 +148,8 @@ class WireNotificationManager @Inject constructor(
                 appLogger.d("$tag; Getting incoming calls for")
                 coreLogic.getSessionScope(userId)
                     .calls
-                    .getIncomingCalls()
-                    .first().also {
+                    .getIncomingCallsOnce()
+                    .also {
```

Comment:
But `.first()` isn't designed to do that ? Get only one item, which the first received, and cancel the flow

Reply:
this PR i kinda "testing", something like "lets try". We have some problems on notifications and looks like it stucks in this `fetchAndShowCallNotificationsOnce`, so I decided to try that :)